boards/renesas/ek_ra4m1/ek_ra4m1_defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
+
+# Enable GPIO
+CONFIG_GPIO=y
+
+# Enable Console
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_CONSOLE=y
+CONFIG_CONSOLE=y
+
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_BUILD_NO_GAP_FILL=y
+CONFIG_CLOCK_CONTROL=y
```

Comment:
these are not board level configs

Reply:
@gmarull : Thank you for your feedback. May I confirm that which level is suitable for these configs? (ex. CONFIG_CLOCK_CONTROL should be in SoC level, CONFIG_BUILD_OUTPUT_HEX ?).

Reply:
I have moved it into SoC configs